packages/components/psammead-caption/src/index.jsx

```diff
@@ -28,13 +28,16 @@ const Caption = styled.figcaption`
   color: ${C_CLOUD_DARK};
   font-family: ${GEL_FF_REITH_SANS};
   ${FS_ITALIC};
-  padding: ${GEL_SPACING} ${GEL_MARGIN_BELOW_400PX};
+  padding: ${GEL_SPACING} ${GEL_MARGIN_BELOW_400PX} 0;
   width: 100%;
   @media (min-width: ${GEL_GROUP_2_SCREEN_WIDTH_MIN}) and (max-width: ${GEL_GROUP_3_SCREEN_WIDTH_MAX}) {
     padding: ${GEL_SPACING} ${GEL_MARGIN_ABOVE_400PX};
```

Comment:
```suggestion
    padding: ${GEL_SPACING} ${GEL_MARGIN_ABOVE_400PX} 0;
```
You're missing the padding-bottom 0 for this breakpoint.